website/content/docs/k8s/installation/vault/data-integration/bootstrap-token.mdx

```diff
@@ -7,11 +7,7 @@ description: >-
 
 # Storing the ACL Bootstrap Token in Vault
 
-## Prerequisites
-Prior to setting up the data integration between Vault and Consul on Kubernetes, you will need to have:
-1. Read and completed the steps in the [Systems Integration](/docs/k8s/installation/vault/systems-integration) section of [Vault as a Secrets Backend](/docs/k8s/installation/vault).
-2. Read the [Data Integration Overview](/docs/k8s/installation/vault/data-integration) section of [Vault as a Secrets Backend](/docs/k8s/installation/vault).
-
+This topic describes how to configure the Consul Helm chart to use an ACL bootstrap token stored in Vault.
 ## Overview
 To use an ACL bootstrap token stored in Vault, we will follow the steps outlined in the [Data Integration](/docs/k8s/installation/vault/data-integration) section:
```

Comment:
```suggestion
The general procedure for storing an ACL bootstrap token stored in Vault is described in the [Data Integration](/docs/k8s/installation/vault/data-integration) section. There are two steps to the procedure. You only need to complete the first two steps once:

1. Store the secret in Vault
1. Create a Vault policy for authorizing access to the secret.
1. For each Consul datacenter, create Vault Kubernetes authorization roles that link the policy to each Consul on Kubernetes service account that requires access.
1. Update the Consul on Kubernetes helm chart. 
```
It took me a minute to understand what's going on. I condensed the following subheadings into an ordered list in this section. If you take this suggestion, then remove the subheadings ("One time setup in Vault" and "Setup per Consul datacenter").

Reply:
ok, but the last two steps are still per data center, so you are updating each data center's helm chart.  This seems harder to understand than:
- do these two things once
- do these two things per data center

which is what the existing format, albeit not the best is trying to convey.

Reply:
I am in favor of the approach already in the existing doc, as it is clearer for a practitioner to understand which is done once and per datacenter with headings. Could we merged these changes as is and perhaps discuss a better way to organize that information afterwards?

Reply:
> ok, but the last two steps are still per data center, you only mention it for step 3. The user is also updating each data center's helm chart. This seems harder to understand than:
> 
> * do these two things once
> * do these two things per data center
> 
> which is what the existing format, albeit maybe not the best format, is trying to convey.

I see what you mean. I updated my suggestion to make that clearer.

Reply:
> I am in favor of the approach already in the existing doc, as it is clearer for a practitioner to understand which is done once and per datacenter with headings. Could we merged these changes as is and perhaps discuss a better way to organize that information afterwards?

I appreciate that I'm not the target audience, but I disagree that the structure is clearer with duplicate headings. Additionally, the use of headings as steps instead of as ways to group related information is inconsistent with our styles. I would prefer to address these issues now, but we can get the content out the door and revisit later.

Reply:
I do see what was done in the new suggestion to apply the change per DC inline. I do think it does require less cognitive load on the user to read instructions per heading as opposed to reading out that a step needs to be done multiple times inline, which may look nicer. My personal opinion would be to optimize for less cognitive load on the user.

Reply:
I think we all want to put the least amount of cognitive load on our readers possible; we just seem to disagree on which approach would accomplish this. Perhaps we should get another opinion?

Reply:
Now that this has been merged, I'll make another branch with the changes to one page that @trujillo-adam  outlined as a Draft PR we can collaborate on.